### PR TITLE
 add devise

### PR DESCRIPTION
**Pull request by naji-97**
- [ ] [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
- [ ] Can register a new user.
- [ ] User logs in with a combination of email and password.
- [ ] Hashed password should be stored in the database.
- [ ] Ask for confirmation of email.
- [ ] Can reset password.
- [ ] [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek]

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (DeliceLydia) [inline app/models/user.rb]: - [ ] Make sure you add `confirmable` so that the application asks for confirmation of email

Comment (DeliceLydia) [reply app/models/user.rb]: - [ ] The `current_user` method in `ApplicationController` should be removed

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Installed devise
- Team members participated
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._





### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @naji-97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**